pdf/extractor/image.go

```diff
@@ -12,10 +12,21 @@ import (
 	"github.com/unidoc/unidoc/pdf/model"
 )
 
+// ImageExtractOpts contains options for controlling image extraction from
+// PDF pages.
+type ImageExtractOpts struct {
```

Comment:
Let's use "Options" rather than "Opts" for consistency (Options used in elsewhere in unidoc).

Reply:
Done